src/main/java/edu/oregonstate/cs361/battleship/BattleshipModel.java

```diff
@@ -76,21 +80,150 @@ public BattleshipModel placeShip(String shipName, String row, String col, String
             }
         }else{
             //vertical
-                if (shipName.equalsIgnoreCase("aircraftcarrier")) {
-                    currModel.getShip(shipName).setLocation(new Coordinate(rowint,colInt),new Coordinate(rowint+5,colInt));
-                } if(shipName.equalsIgnoreCase("battleship")) {
-                    currModel.getShip(shipName).setLocation(new Coordinate(rowint,colInt),new Coordinate(rowint+4,colInt));
-                } if(shipName.equalsIgnoreCase("dinghy")) {
-                    currModel.getShip(shipName).setLocation(new Coordinate(rowint,colInt),new Coordinate(rowint,colInt));
-                } if(shipName.equalsIgnoreCase("clipper")) {
-                    currModel.getShip(shipName).setLocation(new Coordinate(rowint,colInt),new Coordinate(rowint+2,colInt));
-                }if(shipName.equalsIgnoreCase("submarine")) {
-                    currModel.getShip(shipName).setLocation(new Coordinate(rowint, colInt), new Coordinate(rowint + 2, colInt));
-                }
+            if (shipName.equalsIgnoreCase("aircraftcarrier")) {
+                currModel.getShip(shipName).setLocation(new Coordinate(rowint,colInt),new Coordinate(rowint+5,colInt));
+            } if(shipName.equalsIgnoreCase("battleship")) {
+                currModel.getShip(shipName).setLocation(new Coordinate(rowint,colInt),new Coordinate(rowint+4,colInt));
+            } if(shipName.equalsIgnoreCase("dinghy")) {
+                currModel.getShip(shipName).setLocation(new Coordinate(rowint,colInt),new Coordinate(rowint,colInt));
+            } if(shipName.equalsIgnoreCase("clipper")) {
+                currModel.getShip(shipName).setLocation(new Coordinate(rowint,colInt),new Coordinate(rowint+2,colInt));
+            }if(shipName.equalsIgnoreCase("submarine")) {
+                currModel.getShip(shipName).setLocation(new Coordinate(rowint, colInt), new Coordinate(rowint + 2, colInt));
+            }
         }
+
+
+
+        System.out.println("test");
+
+        if ((difficulty == Difficulty.EASY) && (computer_aircraftCarrier.getStart().getAcross() == 0)) {
+            System.out.println("easy mode");
+            computer_aircraftCarrier = new AircraftCarrier(new Coordinate(2, 2), new Coordinate(2, 7));
+            computer_battleship = new Battleship(new Coordinate(2, 8), new Coordinate(6, 8));
+            computer_submarine = new Submarine(new Coordinate(4, 1), new Coordinate(4, 4));
+            computer_clipper = new Clipper(new Coordinate(7, 3), new Coordinate(7, 5));
+            computer_dinghy = new Dinghy(new Coordinate(9, 6), new Coordinate(9, 6));
+        }
+        else if((difficulty == Difficulty.HARD) && (computer_aircraftCarrier.getStart().getAcross() == 0)){
+            System.out.println("hard mode");
```

Comment:
Random place ship in hard mode.
Here you just did similar things for each ship, maybe it could still be shorter.